components/TreasuryAccount/SendTokens.tsx

```diff
@@ -229,7 +229,7 @@ const SendTokens = () => {
         proposalAddress = await createProposal(
           rpcContext,
           realm,
-          selectedGovernance.pubkey,
```

Comment:
this is wrong, we need governance here and not. the token owner. It might be the same but not necessarily